src/api/auth.ts

```diff
@@ -0,0 +1,23 @@
+import { axiosInstance } from './api';
+
+const LOGIN_URL = '/auth';
+
+export interface UserAuthenticationData {
+  email: string;
+  password: string;
+}
+
+const loginUser = (data: UserAuthenticationData): Promise<void> => {
+  const { email, password } = data;
+
+  if (email !== 'test@test.com') throw new Error('Email does not exist!');
```

Comment:
I'm confused about why do we have login. Also why is it hardcoded?

Reply:
It is a skeleton for an API function call with hardcoded error handling to see how the components react. Will add the connection. Also, It would been good to have both access and refresh tokens for JWT authentication. I think currently only access token is available.